langchain/embeddings/openai.py

```diff
@@ -164,5 +135,42 @@ def embed_query(self, text: str) -> List[float]:
         Returns:
             Embeddings for the text.
         """
-        embedding = self._embedding_func(text, engine=self.query_model_name)
+        embedding = self._embedding_func(text, model=self.model)
         return embedding
+
+
+class OpenAIEmbeddings(BaseOpenAIEmbeddings):
```

Comment:
is there any difference between this and the `AzureOpenAIEmbeddings` class besides the docstring?

if not, maybe we dont even need seprate classes, but we can just keep the example thats in the documentation and only add that?

Reply:
No, only the env vars need to be set and it uses same openai package underneath- https://learn.microsoft.com/en-us/azure/cognitive-services/openai/quickstart?pivots=programming-language-python
Let me send update removing separate class

BTW - This is same for completion/llm (langchain.llms.openai.AzureOpenAI) as well, so I thought I would keep it consistent with that. I can help do this later, if you think.

Reply:
@hwchase17 pls see another one https://github.com/hwchase17/langchain/pull/1550 as a follow up